components/page-templates/MyAccountTemplate/MyAccountTemplate.tsx

```diff
@@ -113,6 +63,12 @@ const MyAccountTemplate = (props: MyAccountTemplateProps) => {
   const { updateCustomerAddress } = useUpdateCustomerAddress()
   const { validateCustomerAddress } = useValidateCustomerAddress()
 
+  const [selectedOption, setSelectedOption] = useState('my-profile-content')
+
+  const handleSelectChange = (event: any) => {
+    setSelectedOption(event.target.value)
+  }
+
   const handleGoToOrderHistory = () => {
     router.push('/my-account/order-history?filters=M-6')
```

Comment:
Why this filter is hard coded ?

Reply:
resolve it was from kibo for showing last 6 month order,  removed filter